crowdsec-docs/unversioned/getting_started/installation/kubernetes.mdx

```diff
@@ -102,7 +103,8 @@ If you **have persistentVolumes enabled** in `values.yaml`, you can generate the
 kubectl -n crowdsec exec -it crowdsec-lapi-<pod-id> -- cscli bouncers add my-bouncer-name
 ```
 
-Else you **don't have persistentVolumes enabled**, you need to specify your key in the `values.yaml` file:
+Else you **don't have persistentVolumes enabled**, you need to specify your key
+in the crowdsec helm `values.yaml` file:
 
 ```yaml
```

Comment:
be more verbose on this section:
 - explain what part of the acquis section is specific to kube and what isn't
 - explain link between collections and sources monitored